src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -97,12 +98,21 @@ public function configureOptions(OptionsResolver $resolver)
             'no_more_results_text' => 'No more results',
             'min_characters' => 3,
             'max_results' => 10,
+            'preload' => false,
         ]);
 
         // if autocomplete_url is passed, then HTML options are already supported
         $resolver->setNormalizer('options_as_html', function (Options $options, $value) {
             return null === $options['autocomplete_url'] ? $value : false;
         });
+
+        $resolver->setNormalizer('preload', function (Options $options, $value) {
+            if (\is_bool($value)) {
+                $value = $value ? 'true' : 'false';
+            }
+
+            return $value;
+        });
```

Comment:
Would it work if we removed this and, instead, above, did:

```js
$values['preload'] = json_encode($options['preload']);
```

Reply:
For boolean values it works, but not for strings